### PR TITLE
updated README.md

### PR DESCRIPTION
**Pull request by wymillerlinux**
I filled out most of the README.md in root application directory. There's some stuff that I didn't fill in because we are not at that point yet. Updates coming soon!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,41 @@
...
+### System dependencies: 
+* Bundle: `gem install bundle`
+* Rails: `gem install rails`
+* PostgreSQL: `sudo apt-get install postgresql` OR `sudo dnf install postgresql`
 
-* Database creation
+Windows development machines just need to install the [RailsInstaller](http://railsinstlaler.org) application and [PostgreSQL](http://www.postgresql.org).
 
-* Database initialization
+todo: macOS installation
```
Comment (jtperreault) [inline README.md]: Let's either remove this, or fill it out. The macOS install instructions should be nearly identical. 

1. Install Ruby
2. Install postgreSQL
3. Run `bundle`

File: README.md [hunk 2]
```diff
@@ -1,24 +1,41 @@
...
-* Ruby version
+Some things you want to have:
 
-* System dependencies
+### Ruby version: 
+`2.2.2 or higher`
 
-* Configuration
+### System dependencies: 
+* Bundle: `gem install bundle`
```
Comment (jtperreault) [inline README.md]: Would like this updated to be called it's proper name, '[Bundler](http://bundler.io)'. Though I think `gem install bundle` does in fact get the correct gem. 🤔

File: README.md [hunk 3]
```diff
@@ -1,24 +1,41 @@
...
+Once you have those items installed and working on your machine you will want to run the `bundle` command in the application directory to install the necessary Ruby gems, then run `rails server` to start up the [local development server](http://localhost:3000). After that you should be able to see the application running locally by visiting the page locally.
 
-* Deployment instructions
+## Database creation
 
-* ...
+`bin/rails db:create`
+If that fails, install PostgreSQL.
+
+todo: I totally forgot
```
Comment (jtperreault) [inline README.md]: Please remove todo

File: README.md [hunk 4]
```diff
@@ -1,24 +1,41 @@
...
 
-* ...
+`bin/rails db:create`
+If that fails, install PostgreSQL.
+
+todo: I totally forgot 
+
+## Running the Test Suite
+`bin/rails test`
+
```
Comment (jtperreault) [inline README.md]: Let's remove everything below this line for now. We can add back once we have content.

**Comment by wymillerlinux** [comment]
Updated README per your instructions. School started back up so that is why I haven't touched this for th longest time.

**Pull request merged**